jerry-core/parser/js/common.h

```diff
@@ -52,11 +52,11 @@
 
 /* Malloc functions. */
 
-#define PARSER_MALLOC(size) mem_heap_alloc_block_store_size (size)
-#define PARSER_FREE(ptr) mem_heap_free_block_size_stored ((void *) ptr)
+#define PARSER_MALLOC(size) mem_heap_alloc_block (size)
+#define PARSER_FREE(ptr, size) mem_heap_free_block ((ptr), (size))
 
-#define PARSER_MALLOC_LOCAL(size) mem_heap_alloc_block_store_size (size)
-#define PARSER_FREE_LOCAL(ptr) mem_heap_free_block_size_stored (ptr)
+#define PARSER_MALLOC_LOCAL(size) mem_heap_alloc_block (size)
+#define PARSER_FREE_LOCAL(ptr, size) mem_heap_free_block ((ptr), (size))
```

Comment:
Do we still need to keep these macros?
- They seem to duplicate each other.
- Their only valid uses seem to be in js-parser-mem.c.
- This patch already replaces them at some points.

I'd vote for replacing them with calls to `parser__{malloc,free}[_local]` or `mem_heap_{alloc,free}_block`, depending on the use case.